### DISCUSSION TITLE
Seeking Feedback for Our WhatsApp Webhook Repository: Let's Make it Even Better!

### DISCUSSION THREAD

**Discussion by junedkhatri31** [whatsapp-webhook/whatsapp-webhook#2] [Ideas]
Hello!

We're excited to share our open-source 'WhatsApp Webhook' repository with the community. We've created a basic app that displays messages sent by users, and now we're eager to hear from you! What kind of features would you like to see in the future releases of this project? We would also love to know why you're here in this Github repository. Your input and feedback will help us make this project even better. Let us know your thoughts in the comments section below.

Thank you for your support!

**Comment by kfenech1** [comment]
Hi, this is an awesome project. Thumbs up and I would love to see more development on it.

It would be nice to see:
- being able to receive audio/video/image messages from the users
- bulk sending messages to users

I am not much of a javascript and typescript developer myself but if I can help in anyway I would love to. 

I am here in this project as I was looking for an open source and simple to use webhook receiver

**Reply by junedkhatri31** [reply]
Thanks @kfenech1 . I am working on bulk sending. Will be out soon. Stay tuned. Also this project is now moved to https://github.com/receevi/receevi. Please take a look at the new project.

**Reply by kfenech1** [reply]
Ok thanks @junedkhatri31 :) I managed to get it up and running now after using the new project. Looking forward to the bulk send feature.

One suggestion. There is a nice integration between supabase and vercel. If you change the name of the env vars in the project to match the automatically added env vars when you sent the integration that would be great. I will try to have a look at the code some time next week and open an MR in the other project

**Reply by junedkhatri31** [reply]
Thanks for the suggestion. I will look into Supabase and Vercel integration. I will make self hosting also easier.
